### ISSUE TITLE
kNN classification on pretrained checkpoints

### ISSUE THREAD

**Issue by bourcierj** [aicip/Cross-Scale-MAE#1]
Hi,

I am trying to use the provided ViT-Large checkpoint of Cross-Scale-MAE to perform inference and kNN classification on features. When I test it on the RESISC45 dataset, I get relatively low accuracy (0.6211). I expect it to be much higher given your results in the paper.

Could you provide the expected performance on RESISC45 at 100% scale ratio, for comparison ? This would help for debugging and give me a target to reproduce.

Some details: I am using the RESISC45 train/test splits from torchgeo, image size = 256 (which should correspond to 100% scale ratio I believe), k = 5.

Thank you.
